src/pages/Touringcar/components/ViewerContainer.tsx

```diff
@@ -42,7 +42,7 @@ export const TouringcarViewerContainer = ({ showDesktopVariant, ...otherProps }:
   return (
     <>
       {!showDesktopVariant ? (
-        <StyledViewerContainer {...otherProps} ignoreTransition={draggable} />
+        <StyledViewerContainer {...otherProps} ignoreTransition={draggable} topRight={<TouringcarMapLegend />} />
```

Comment:
We might want to remove the outer showDesktopVariant check and combine these two `StyledViewerContainer`'s. I have a feeling that it will improve readability. What do you think? We can check that showDesktopVariant for the specific properties.

```
{!showDesktopVariant ? (
        <StyledViewerContainer {...otherProps} ignoreTransition={draggable} topRight={<TouringcarMapLegend />} />
      ) : (
        <StyledViewerContainer
          {...otherProps}
          ignoreTransition={draggable}
          bottomRight={<Zoom />}
          topRight={<TouringcarMapLegend />}
        />
      )}
```